release/README.md

```diff
@@ -8,40 +8,53 @@ https://semver.org/
 1. To release a major or minor version create a release branch `release-X.Y` from the last minor release tag.
 
     ```bash
-    git checkout vA.B.0
+    git checkout main
     git checkout -b release-X.Y
     git push -u origin release-X.Y
     ```
 
-2. Checkout the branch to cut the release from.
+1. Reset changelog
 
     ```bash
-    git checkout main
-    git pull
-    git checkout -b branch_name
+    git checkout -b reset-changelog-X.Y
+    release/reset-changelog.sh X.Y.0
     ```
 
-3. Run QA checks on `branch_name` and commit any fixes found during QA.
+    The release script will:
+    * Append what is in `WIP-CHANGELOG.md` to `CHANGELOG.md`
+    * Clear `WIP-CHANGELOG.md`
+    * Create a git commit with message `Release vX.Y.Z`
```

Comment:
Won't this look strange if we add fixes after it while doing QA?

Git log:
```
321cba (tag: v1.0) Fix issue found in QA <-- The tagged release
abc123 Release v1.0 <-- from resetting the changelog
```

Reply:
Yea, that's true. Maybe we could change the message to something like `Cut for release vX.Y.Z ` and add then add another commit after QA which the tag is set to and that has the message `Release vX.Y.Z`

Reply:
Sounds good! Maybe rather `Clear WIP-CHANGELOG.md` or `Clear WIP-CHANGELOG.md for vX.Y.Z`  as message?

Reply:
Apparently I've already done this in the script but just forgot to change here
https://github.com/elastisys/compliantkubernetes-apps/pull/306/files#diff-a36345fe16d00b329e46048d709e1a54a6fea61245d0f56bef4a91b4e053e6ccR68
But I should fix the release script so it adds the release commit.